src/exercises/chromium/third-party.md

```diff
@@ -9,13 +9,23 @@ skip ahead and do that now if you like. Or, you could create a new
 [`rust_executable` target][2] which uses `uwuify`).
 
 <details>
+
 Students will need to download lots of transitive dependencies.
 
-The total crates needed are: uwuify, smallvec, scopeguard, parking_lot,
-parking_lot_core, lock_api and instant. If students are downloading even
+The total crates needed are:
+instant,
+lock_api,
+parking_lot,
+parking_lot_core,
+redox_syscall,
+scopeguard,
+smallvec,
+and uwuify.
+If students are downloading even
```

Comment:
The formatting here will disappear as soon as we start auto-formatting the Markdown code.

If you want a sortable list, I would suggest using a real Markdown list:

```suggestion
The total crates needed are:

- `instant`,
- `lock_api`,
- `parking_lot`,
- `parking_lot_core`,
- `redox_syscall`,
- `scopeguard`,
- `smallvec`, and
- `uwuify`.

If students are downloading even
```

Notice that the `_` above should be put into `` `...` `` like in the suggestion: otherwise they can trigger _emphasis_ (I think it depends a bit on the precise rules of the Markdown implementation if they actually trigger emphasis.)

Reply:
Done